### PR TITLE
Form login logout

### PR DESCRIPTION
**Pull request by PechBunsinh**
- Create Users Controller
- Add Link path in routs file
- Create Form Signup
- Create Form Login
- Remove test directory
- Remove users.coffee file

### CODE REVIEW

File: pj_movie/app/assets/stylesheets/admin/top_page.scss [hunk 1]
```diff
@@ -0,0 +1,238 @@
...
+            text-align: start;
+          }
+        }
+        //-------------dropdown_content-----------------
+        .dropdown_content {
+          margin-top: 12px;
+          position: absolute;
+          background-color: white;
+          width: 245.98px;
+          //box-shadow: 0px 8px 16px 0px rgba(0,0,0,0.2);
```
Comment (loemhuolay) [inline pj_movie/app/assets/stylesheets/admin/top_page.scss]: You should remove this un-use line

File: pj_movie/app/assets/stylesheets/admin/top_page.scss [hunk 2]
```diff
@@ -0,0 +1,238 @@
+// Place all the styles related to the admin/TopPage controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: http://sass-lang.com/
```
Comment (loemhuolay) [inline pj_movie/app/assets/stylesheets/admin/top_page.scss]: These line should be delete.

File: pj_movie/app/assets/stylesheets/admin/users.scss [hunk 3]
```diff
@@ -0,0 +1,9 @@
+// Place all the styles related to the admin/Users controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: http://sass-lang.com/
+/* forms */
```
Comment (loemhuolay) [inline pj_movie/app/assets/stylesheets/admin/users.scss]: These line also should delete.

File: pj_movie/app/assets/stylesheets/reset.css [hunk 4]
```diff
@@ -0,0 +1,102 @@
+/*
+html5doctor.com Reset Stylesheet
+v1.6.1
+Last Updated: 2010-09-17
+Author: Richard Clark - http://richclarkdesign.com
+Twitter: @rich_clark
+*/
+
```
Comment (loemhuolay) [inline pj_movie/app/assets/stylesheets/reset.css]: These line should be delete.

File: pj_movie/app/assets/stylesheets/admin/users.scss [hunk 5]
```diff
@@ -0,0 +1,5 @@
+.admin_signin {
+  margin-top: 98px;
+
```
Comment (N310) [inline pj_movie/app/assets/stylesheets/admin/users.scss]: you should remove this empty line.

File: pj_movie/app/views/admin/users/login.html.erb [hunk 6]
```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html>
+  <head>
+  <title><%= yield(:title) %> | Project Movie</title>
+  <%= render 'layouts/rails_default' %>
+  <%= render 'layouts/shim' %>
+  </head>
+<body>
```
Comment (N310) [inline pj_movie/app/views/admin/users/login.html.erb]: you should delete this lines.

File: pj_movie/app/views/admin/users/login.html.erb [hunk 7]
```diff
@@ -0,0 +1,30 @@
...
+
+            <%= f.submit "Log in", class: "btn_signup" %>
+
+            <p><%= link_to "Foget Password?", "#" %><p>
+          <% end %>
+        </div>
+      </form>
+    </div>
+  </dvi>
+</body>
```
Comment (N310) [inline pj_movie/app/views/admin/users/login.html.erb]: you should delete this lines.

**Comment by N310** [comment]
Why don't you create sing up page for admin?

File: pj_movie/app/views/admin/users/new.html.erb [hunk 8]
```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html>
+  <head>
+  <title><%= yield(:title) %> | Project Movie</title>
+  <%= render 'layouts/rails_default' %>
+  <%= render 'layouts/shim' %>
+  </head>
+<body>
```
Comment (N310) [inline pj_movie/app/views/admin/users/new.html.erb]: you should delete this lines.

File: pj_movie/app/views/admin/users/new.html.erb [hunk 9]
```diff
@@ -0,0 +1,33 @@
...
+            <%= f.label :password_confirmation, "Confirm Password" %>
+            <%= f.password_field :password_confirmation, class: 'form-control', :required => true %>
+
+            <%= f.submit "Sign up", class: "btn_signup" %>
+          <% end %>
+        </div>
+      </form>
+    </div>
+  </div>
+</body>
```
Comment (N310) [inline pj_movie/app/views/admin/users/new.html.erb]: you should delete this lines.

**Pull request merged**